.github/hs-deploy-action/src/deploy.ts

```diff
@@ -17,16 +17,27 @@ export async function deploy(github: any, octoKit: any) {
   const command = `upp application deploy -b ${bundleId} -t spartacus -s ./dist/storefrontapp -e stage`;
 
   const exp = /https\:\/\/\w+\.cloudfront\.net/;
+  const ERROR = 'Response code: 500';
 
   const options: ExecOptions = {};
   options.listeners = {
-    stdout: (data: Buffer) => {
+    stdout: async (data: Buffer) => {
       const line = data.toString();
+
+      if (line.includes(ERROR)) {
+        const body = `:exclamation: Spartacus deployment failed (${line}). Check job logs for details.)`;
```

Comment:
emoji doesn't seem to be working in the logs (or at least in this format)

Reply:
They probably don't support it. But it was meant for the comment on github.